### PR TITLE
add(angular): add blocker for angular meetup

### LINKED ISSUE #85: Blocker for Angular Berlin meetup 29.5.2018
We would like to host an event on Tuesday the 29.th of May. Details are following

### PR DESCRIPTION
**Pull request by kotzendekrabbe**
fix #85

### CODE REVIEW

File: _posts/2018-05-29-angular.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+layout: post
+title: "Agnular Meetup"
```
Comment (ghost) [inline _posts/2018-05-29-angular.md]: this should say Angular

**Review by ghost — CHANGES_REQUESTED** [review]

File: _posts/2018-05-29-angular.md (hunk 1 above)
Comment (kotzendekrabbe) [reply _posts/2018-05-29-angular.md]: oh yes. thanks so much :)

**Comment by kotzendekrabbe** [comment]
fixed :) @oe

**Review by ghost — APPROVED** [review]
LGTM!

**Pull request merged**